kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/preprocess/FieldNameSanitizer.java

```diff
@@ -0,0 +1,31 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FieldNameSanitizer {
+
+    private static String sanitizeName(String name) {
+        name = name.replaceAll("^[^a-zA-Z_]+", "");
+        return name.replaceAll("[^a-zA-Z0-9_]", "_");
+    }
+
+
+    // Big Query specifies field name must begin with a letter or underscore and can only contain
+    // letters, numbers, and underscores
```

Comment:
we should put this as a javadoc either on the method above or on the class, to make it clear what the regex refer to